<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="databaseEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImagesButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagesEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDictionaryButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dictionaryEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chooseDescriptorlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Descriptor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT and LBP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT and OC-LBP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT, LBP, HUE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SIFT, OC-LBP, HUE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HOG</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HOG and LBP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>radioBtnSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>radioBtnSelected()</slot>
 </slots>
</ui>
